File: bluetoothProject/app/src/main/java/com/example/bluetoothproject/WaterTankActivity.java
package com.example.bluetoothproject;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class WaterTankActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_water_tank);

        ImageButton home2 = (ImageButton)findViewById(R.id.home2);
        ImageButton setting2 = (ImageButton) findViewById(R.id.setting2);

        home2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                finish();
            }
        });

        setting2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                /*
                Intent intent = new Intent(getApplicationContext(), SettingActivity.class);
                startActivity(intent);

                 */
            }
        });

    }
}

File: PlantEngine_TermProject/app/src/main/java/com/example/plantengine_termproject/WaterTankActivity.kt
package com.example.plantengine_termproject

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_water_tank.*

class WaterTankActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_water_tank)

        ClickLister();

    }
    private fun ClickLister(){
        home2.setOnClickListener {
            finish()
        }
        setting2.setOnClickListener {
            val intent: Intent = Intent(this, SettingActivity::class.java)
            startActivity(intent)
        }
    }
}